modules/Kconfig.stm32

```
@@ -550,6 +550,11 @@ config USE_STM32_LL_DAC
 	  Enable STM32Cube Digital-to-analog converter (DAC) LL module driver
 
 config USE_STM32_LL_DELAYBLOCK
+	bool
+	help
+	  Enable STM32Cube DelayBlock (DELAYBLOCK) LL module driver (stm32H7 or stm32MP1)
+
+config USE_STM32_LL_DLYB
 	bool
 	help
 	  Enable STM32Cube DelayBlock (DELAYBLOCK) LL module driver
```

The `USE_STM32_LL_DELAYBLOCK` help contains the series name, maybe it is worth adding it to `USE_STM32_LL_DLYB` as well?

```suggestion
	  Enable STM32Cube DelayBlock (DELAYBLOCK) LL module driver (stm32U5)
```

---

modified